### ISSUE TITLE
VO-DML uses "domain" in a confusing way

### ISSUE THREAD

**Issue by olebole** [ivoa-std/ProvenanceDM#5]
The VO-DML for `Entity` has the following description:
> A thing which is consumed or produced in a task , namely here in the astronomical domain. It represents some data as single element or file or an instrument.

The description of `Entity.id` has the following description: 
> Unique identifier for an Entity. Must be unique in the domain.

This may imply that the `Entity.id` must be unique within the whole astronomical domain, which is probably not what is meant here. However, there is no other definition of "domain" throughout the document. Therefore, I would propose to remove the second sentence in the description of `Entity.id` and make it identical to the Table 1 of the PDF:
> **id** | string | a unique identifier for this entity

For convenience, here the lines in the model: https://github.com/ivoa-std/ProvenanceDM/blob/0f4853f876de0c6aec9e4d723593d02b3ec218a2/model/Provenance.vo-dml.xml#L401-L412

**Comment by lmichel** [comment]
Ole,
This modification on a standards must follow the errata process (https://www.ivoa.net/documents/DocStd/20170517/REC-DocStd-2.0-20170517.pdf p12). 
I've no idea about how does this fit with the Gihub workflow, but the errata must be accepted by the TCG before issuing any modification in the document.

**Comment by olebole** [comment]
The handling here is IMO two-fold: First, a patch should be created, reviewed, and merged into the document repository; the original version remains the "to be tagged one", see #4 (pinging you and @mservillat again…). I volunteer to create the patch here. This step would make clear that the raised issue is relevant, and it will be fixed with the next release. I do not see the need to involve the TCG until then.

Once this is done, the process documented in [IVOA Document Standards](https://www.ivoa.net/documents/DocStd/20170517/REC-DocStd-2.0-20170517.pdf) can be started and may lead to an Errata.

Does this sound reasonable?

**Comment by lmichel** [comment]
The erratum process goes through a Wiki page, no need to do patch.
I can set it up (after interop).
As you are an author, I believe that all the others have to agree with that erratum.

**Comment by mservillat** [comment]
It is said in the standard, section 1.2.1, that "Entities, Activities and Agents must be uniquely identifiable within a domain."
This fixes the context of those definitions, "the domain" is the one chosen for this entity. In general, I am reluctant to prepare an erratum if there is not much to write in the Rationale and Impact Assessment sections of the erratum. The phrasing may not be optimal, but the impact seems extremely low.

**Comment by olebole** [comment]
@mservillat The context is completely different: 1.2.1 is

> ### 1.2.1 Model requirements
> This document was developed with these general requirements in mind:
> * […]
> * Entities, Activities and Agents must be uniquely identifiable within a domain.

This does neither say what the "domain" is here (one of the flaws of the document), nor does it say how this uniqueness is reached, especially not about which fields are evaluated here. And it is a requirements *to the model* (the standard), not to an implementation; we split the latter apart in 1.2.2 (remember the discussion about this?).

And, the problem here is the model, which does not contain this phrase at all. The model puts "the domain" next to "the astronomical domain", implying that this one is meant, which is just wrong. And the solution is simply to synchronize the text here with the one from the printed document (which is recommended anyway).

**Comment by olebole** [comment]
@lmichel This should be changed in the model for the next version independently whether this is an Erratum or not.

**Comment by lmichel** [comment]
If this should be changed in the model for the next version independently whether this is an Erratum or not, the best is to let the issue open until you will be working on this upgrade.
This is a very minor issue that does not lead any sort of confusion. Therefore that does not  justify to open an erratum and to bother the TVG to validate it.

**Comment by olebole** [comment]
@lmichel so we can just merge the PR #6?

**Comment by lmichel** [comment]
As I said we are not supposed to amend a REC once it has been approved.

**Comment by olebole** [comment]
The REC is already tagged and uploaded -- this is what the tags are for (see #4). However, for future versions, we should fix problems as they appear, and this is what the repository is for.
So, with the merge, the REC version stays unchanged, it just adds the change for a future release.
